packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -1,413 +1,162 @@
-                { item.name }
-              </span>
-            ) }
-            { hasSubmenuItems(item) ? (
-              <Icon className={ css('ms-CommandBarItem-chevronDown', styles.itemChevronDown) } iconName='chevronDown' />
-            ) : (null) }
-          </button>;
-        } else if (item.href) {
-          return <a
-            { ...getNativeProps(item, anchorProperties) }
-            id={ this._id + item.key }
-            className={ className }
-            href={ item.href }
-            data-command-key={ index }
-            aria-haspopup={ hasSubmenuItems(item) }
-            role='menuitem'
-            aria-label={ item.ariaLabel || item.name }
-          >
-            { (hasIcon) ? this._renderIcon(item) : (null) }
-            { (!!item.name) && (
-              <span
-                className={ css('ms-CommandBarItem-commandText', styles.itemCommandText) }
-              >
-                { item.name }
-              </span>
-            ) }
-          </a>;
-        } else {
-          return <div
-            { ...getNativeProps(item, divProperties) }
-            id={ this._id + item.key }
-            className={ className }
-            data-command-key={ index }
-            aria-haspopup={ hasSubmenuItems(item) }
-          >
-            { (hasIcon) ? this._renderIcon(item) : (null) }
-            <span className={ css(
-              'ms-CommandBarItem-commandText', styles.itemCommandText) } aria-hidden='true' role='presentation'>{ item.name }</span>
-          </div>;
-        }
-      })() }
-    </div>;
-  }
-
-  private _renderIcon(item: IContextualMenuItem) {
-    // Only present to allow continued use of item.icon which is deprecated.
-    let iconProps: IIconProps = item.iconProps ? item.iconProps : {
-      iconName: item.icon as IconName
-    };
-    // Use the default icon color for the known icon names
-    let iconColorClassName = iconProps.iconName === 'None' ? '' : ('ms-CommandBarItem-iconColor ' + styles.itemIconColor);
-    let iconClassName = css('ms-CommandBarItem-icon', styles.itemIcon, iconColorClassName, iconProps.className);
-
-    return <Icon { ...iconProps } className={ iconClassName } />;
-  }
-
-  private _updateItemMeasurements() {
-    // the generated width for overflow is 35 in chrome, 38 in IE, but the actual value is 41.5
-    if (this.refs[OVERFLOW_KEY] || (this.props.overflowItems && this.props.overflowItems.length)) {
-      this._overflowWidth = OVERFLOW_WIDTH;
-    } else {
-      this._overflowWidth = 0;
-    }
-
-    if (!this._commandItemWidths) {
-      this._commandItemWidths = {};
-    }
-
-    for (let i = 0; i < this.props.items.length; i++) {
-      let item = this.props.items[i];
-
-      if (!this._commandItemWidths[item.key]) {
-        let el = this.refs[item.key] as HTMLElement;
-
-        if (el) {
-          this._commandItemWidths[item.key] = el.getBoundingClientRect().width;
-        }
-      }
-    }
+    // this.refs.focusZone.focus();
```

Comment:
yup :) need to dig into what this did in the original so that I can replicate it.